includes/collection/class-extra-fields.php

```diff
@@ -39,6 +39,12 @@ public static function get_actor_fields( $user_id ) {
 			$args['author'] = $user_id;
 		}
 
+		// Limit to 100 fields to prevent response size issues.
+		if ( ! is_admin() ) {
+			$args['nopaging']       = false;
+			$args['posts_per_page'] = 100;
```

Comment:
```suggestion
			$args['posts_per_page'] = 20;
```

I would go even lower!

Reply:
Fixed in [8724146](https://github.com/Automattic/wordpress-activitypub/pull/1660/commits/8724146796ce01630453f72987bd8d81dc3de6e6).